lib/seeds/course_seeder.rb

```diff
@@ -26,15 +28,19 @@ def add_section(&)
       end
     end
 
+    # rubocop:disable Metrics/AbcSize
```

Comment:
Over the score by about 2. A bit annoying, but I don't think breaking this up particularly improves it. Just a bit of a data clump.